osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/SetConfigurationObjectService.java

```diff
@@ -0,0 +1,125 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.openmuc.jdlms.AccessResultCode;
+import org.openmuc.jdlms.AttributeAddress;
+import org.openmuc.jdlms.SetParameter;
+import org.openmuc.jdlms.datatypes.BitString;
+import org.openmuc.jdlms.datatypes.DataObject;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.utils.DlmsHelper;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.utils.JdlmsObjectToStringUtil;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.factories.DlmsConnectionManager;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ConnectionException;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.ConfigurationFlagDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.ConfigurationFlagTypeDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.ConfigurationObjectDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class SetConfigurationObjectService implements ProtocolService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SetConfigurationObjectService.class);
+    private static final int NUMBER_OF_FLAG_BITS = 16;
+    private static final int BYTE_SIZE = 8;
+
+    private final DlmsHelper dlmsHelper;
+
+    SetConfigurationObjectService(final DlmsHelper dlmsHelper) {
+        this.dlmsHelper = dlmsHelper;
+    }
+
+    public AccessResultCode setConfigurationObject(final DlmsConnectionManager conn,
+            final ConfigurationObjectDto configurationToSet, final ConfigurationObjectDto configurationOnDevice)
+            throws ProtocolAdapterException {
+
+        final DataObject dataObject = this.buildSetParameterData(configurationToSet, configurationOnDevice);
+        LOGGER.info("ConfigurationObject SetParameter Data : {}", this.dlmsHelper.getDebugInfo(dataObject));
```

Comment:
should this be debug or info ?

Reply:
Apparently we want to log debugInfo from the dlmsHelper at INFO level.

Reply:
but why? in the world. a lot of debug code is added as info because most app servers run on info as lowest level.

Reply:
Ok, I will lower the level to DEBUG (or remove it completely) after the feature is tested succesfully.